conf/docker-aio/c8.dockerfile

```diff
@@ -0,0 +1,82 @@
+FROM centos:8
```

Comment:
Since we're creating this new c8.dockerfile, should we go ahead and delete the c7.dockerfile in this directory?

Reply:
It might be useful for testing on cent7 and cent8; but that's probably better handled through dataverse-ansible.

Reply:
I removed the old one. We can always pull it out of git history if we need it.